### ISSUE TITLE
Restore snapshot fails

### ISSUE THREAD

**Issue by zeventh** [LINBIT/drbdmanage-proxmox#2]
Create a snapshot from Proxmox web gui works fine and we can list the snapshot with "drbdmanage list-snapshots" but when we try to restore the snapshot it fails with the message "snapshot restore is not implemented yet".

**Comment by rck** [comment]
Yes, this is known, and this is exactly why it prints that message. The way Proxmox wants to restore snapshots (in place) and the way DM does it (creating a new DRBD resource) do not match. Eventually this will be implemented, but don't expect anything in the next days, probably not even in the next months.

**Comment by fab33** [comment]
and the next year ? ;) It will be a very good feature

**Comment by rck** [comment]
@fab33 This project is basically EoL, as the underlying technology (drbdmanage) is basically EoL. The successor is called LINSTOR. While it has not reached a 1.0 yet, it already works a lot better than drbdmanage. It also has a Proxmox plugin that is under active development (e.g., it will get support for multiple storage pools in the next days).

Unfortunately, the fundamental differences between how we restore snapshots and the expected rollback remains.

tl;tr: forget drbdmanage and move to LINSTOR. Don't expect restore there.
